README.md

```diff
@@ -47,21 +46,20 @@ In this case, the output will be:
 {'00': 513, '11': 511}
 ```
 
-A script is available [here](examples/python/ibmq/hello_quantum.py), where we also show how to
-run the same program on a real quantum computer via IBMQ.
+A script is available [here](examples/python/ibmq/hello_quantum.py), where we also show how to run the same program on a real quantum computer via IBM Quantum.
 
 ### Executing your code on a real quantum chip
 
 You can also use Qiskit to execute your code on a
 **real quantum chip**.
-In order to do so, you need to configure Qiskit for using the credentials in
-your IBM Q account:
+In order to do so, you need to configure Qiskit with credentials for using a quantum computer cloud provider. For example, to connect it to your IBM Quantum account:
```

Comment:
For this section I'd actually make it more general and less IBM specific at the beginning and only mention IBM as an example. Something like:

```suggestion
Qiskit enables you to run code on real quantum computers. To to do this it provides an abstraction
layer that lets hardware vendors build packages that provde an interface to their hardware from Qiskit.
Using these ``providers`` you can run any Qiskit code against a real quantum computer.

For example, to use Qiskit with IBM Quantum's cloud quantum computing service:
```